tests/unit/config/test_configinit.py

```diff
@@ -65,6 +66,20 @@ def configdata_init(monkeypatch):
 
 class TestEarlyInit:
 
+    def test_config_py_path(self, args, tmpdir, init_patch):
+        f = tmpdir.join('temp_config.py')
+        f.write('c.colors.hints.bg = "red"\n')
+        args.config_py = str(f)
+        configinit.early_init(args)
+
+        # Make sure things have been init'ed
```

Comment:
@jgkamat Not sure if you've seen it already:

https://github.com/qutebrowser/qutebrowser/pull/4286#discussion_r222755876
> I don't think those two tests are needed here.